doc/lua_api.md

```diff
@@ -7719,13 +7723,19 @@ child will follow movement and rotation of that bone.
         * `acceleration_air`: multiplier to acceleration
           when jumping or falling (default: `1`)
             * Note: The actual acceleration is the product of `speed` and `acceleration_air`
+        * `acceleration_fast`: multiplier to acceleration in Fast Mode (default: `1`)
+            * Note: The actual acceleration is the product of `speed` and `acceleration_fast`
         * `sneak`: whether player can sneak (default: `true`)
         * `sneak_glitch`: whether player can use the new move code replications
           of the old sneak side-effects: sneak ladders and 2 node sneak jump
           (default: `false`)
         * `new_move`: use new move/sneak code. When `false` the exact old code
           is used for the specific old sneak behavior (default: `true`)
     * Note: All numeric fields above modify a corresponding `movement_*` setting.
+    * Note: `speed` and the other `speed_*` modifiers will be multiplied with
+      each other. Example: If `speed=2`, `speed_walk=3` and everything else is default,
+      walking speed factor is 6, while climbing, sneaking and Fast Mode speed has
+      a factor of 2.
```

Comment:
Is another note about this really necessary? There's already a note in the documentation for `speed` itself, and a note in the documentation for each of the affected overrides. If you want to keep this note:

- This doesn't only apply to `speed_*` overrides, but also to `acceleration_*` overrides.

- "will be multiplied with each other" seems confusing, since `speed` doesn't change.

- "If `speed=2`, `speed_walk=3` and everything else is default": I think writing this out as Lua code would be more understandable.

    ```lua
    player:set_physics_override({
        speed = 2,
        speed_walk = 3,
    })
    ```